automation/roles/postgresql-privs/tasks/main.yml

```diff
@@ -0,0 +1,22 @@
+---
+
+- name: Grant/revoke privileges on objects
+  community.postgresql.postgresql_privs:
+    roles: "{{ item.role }}"
```

Comment:
```suggestion
    roles: "{{ item.role | default('postgres') }}"
```

Reply:
Hi @vitabaks that setting `"{{ item.db | default('postgres') }}"` should be ok since objs is required always in this role, that `"{{ item.role | default('postgres') }}"` could assign permissions to the wrong user, so `- item.role | default('') | length > 0` should be enough. What do you think?

Reply:
Ok, new [comment](https://github.com/vitabaks/postgresql_cluster/pull/740#discussion_r1738163943)